tests/test_idtools.py

```diff
@@ -0,0 +1,54 @@
+from langworld_db_data.idtools.value_id_tools import (
+    _split_value_id,
+    extract_category_id,
+    extract_feature_id,
+    extract_feature_index,
+    extract_value_index,
+)
+
+IDS_FOR_EXTRACTION = [
+    "A-1-2",
+    "B-7-22",
+    "C-12-6",
+    "K-14-26",
+]
```

Comment:
```suggestion
IDS_FOR_EXTRACTION = (
    "A-1-2",
    "B-7-22",
    "C-12-6",
    "K-14-26",
)
```
let's make it explicitly immutable

Reply:
same for other iterables below

Reply:
A list of lists could have become a tuple of tuples, not a tuple of lists, as you did, but again, it's not important